### ISSUE TITLE
Seeking maintainers

### ISSUE THREAD

**Issue by dylanmei** [packer-community/winrmcp#38]
The **winrmcp** project and the **packer-community** organization began in 2015 by a few intrepid engineers collaborating to automate their way out of some gnarly infrastructure problems. As one does.

Each of us no longer uses this software, and are unable to provide proper validation and guidance for patches and improvements to **winrmcp**, or any of the projects under the **packer-community** umbrella.

If you have a genuine interest in solving the same problems we were solving and can carry the torch onward, or have an idea to gracefully decommission **winrmcp** and could shepherd this process, I would like to hear about it and help make this happen.

**Comment by johnypony3** [comment]
8 years without supporting and with out hashicorp dealing with this.

Please sunset it so packers users aren't working your product.

Thanks.

:killitwithfire:

**Comment by dylanmei** [comment]
Kindly explain "sunset it."

**Comment by johnypony3** [comment]
Decom, sunset. At the very least update readme and let hashicorp know its not being maintained. Not much else you can do besides finding a replacement, maybe @mwrock suggested... checks clipboard... [7 years ago](https://github.com/packer-community/winrmcp/pull/6#issuecomment-171345437).

I cant imagine not dealing with an mr for 8 years. I guess what I'm asking you to do is something.

**Comment by arizvisa** [comment]
I'm pretty sure the author pretty much put in enough effort attempting that route. Best thing to do is just delete the repo so all the things like hashicorp/packer-plugin-sdk that depend on `winrmcp.Winrmcp` can't be built anymore. At that point it'll stimulate a rewrite and then with luck transfers of large files won't be racy.

**Comment by johnypony3** [comment]
Yep, agreed. Delete, make private, whatever. I don't have enough context to know what was and wasn't done by this author. I do know the pr in question should have been merged 8 years ago.

**Comment by dylanmei** [comment]
This issue is a call for maintainers. To discuss a winrmcp sunset, visit #39

**Comment by anurag5sh** [comment]
I'm keen to get some of the improvement PRs merged since they can benefit Packer significantly. I'm also happy to step in and take on the maintainer role to help keep things moving forward.

**Comment by robinsingh1** [comment]
Hi @dylanmei , would be interested in helping maintain the project - what could potential next steps be ?
